docs/team/yucheng11122017.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: Chan Yu Cheng's Project Portfolio Page
+---
+
+### Project: CONNECTIONS
+
+CONNECTIONS is a desktop address book application used for teaching Software Engineering principles.
```

Comment:
u think we should describe our product properly instead of using the description for AB3? bc AB3 is indeed an app to teach SWE but ours isn't imo

Reply:
Not too sure about this, but if u do change it then I will probably change mine too.